libbeat/tests/system/test_dashboard.py

```diff
@@ -36,6 +36,32 @@ def test_load_dashboard(self):
 
         assert self.log_contains("Kibana dashboards successfully loaded") is True
 
+    @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
+    @attr('integration')
+    def test_load_only_index_patterns(self):
+        """
+        Test loading dashboards
+        """
+        self.render_config_template()
+        beat = self.start_beat(
+            logging_args=["-e", "-d", "*"],
+            extra_args=["setup",
+                        "--dashboards",
+                        "-E", "setup.dashboards.file=" +
+                        os.path.join(self.beat_path, "tests", "files", "testbeat-dashboards.zip"),
+                        "-E", "setup.dashboards.beat=testbeat",
+                        "-E", "setup.dashboards.only_index=true",
+                        "-E", "setup.kibana.protocol=http",
+                        "-E", "setup.kibana.host=" + self.get_kibana_host(),
+                        "-E", "setup.kibana.port=" + self.get_kibana_port(),
+                        "-E", "output.elasticsearch.hosts=['" + self.get_host() + "']",
+                        "-E", "output.file.enabled=false"]
+        )
+
+        beat.check_wait(exit_code=0)
+
+        assert self.log_contains("Kibana dashboards successfully loaded") is True
```

Comment:
I agree it's confusing that the loading index patterns and dashboard are conflated but I think it would be best to tackle that as an enhancement rather than in this bugfix, that I'm hoping we can backport to 6.2

Reply:
Got it, agree. I added the `needs_backport` label. Could you open a follow up PR with the change mentioned above?

Reply:
Will do, thanks @ruflin